qtpy/QtGui.py

```diff
@@ -72,3 +72,12 @@ def movePositionPatched(
     ) -> bool:
         return movePosition(self, operation, mode, n)
     QTextCursor.movePosition = movePositionPatched
+
+# close https://github.com/spyder-ide/qtpy/issues/394
```

Comment:
```suggestion
# Fix https://github.com/spyder-ide/qtpy/issues/394
```

Here we're not actually closing the issue (like e.g. in a PR body), but rather trying to fix it.

Reply:
There is no consistent constructor for the event. I mean the constructor changed over the years. Hence, I can compose either separate constructors for Qt versions 5.12, 5.15, and 6.3, or make the event appear naturally. I'm to try the latter approach first.